src/main/java/io/strimzi/kafka/quotas/VolumeSource.java

```diff
@@ -118,28 +148,60 @@ private CompletionStage<VolumeUsageResult> onDescribeClusterComplete(Result<Coll
 
     private CompletionStage<VolumeUsageResult> onDescribeClusterSuccess(Collection<Node> nodes) {
         final Set<Integer> allBrokerIds = nodes.stream().map(Node::id).collect(toSet());
+        activeBrokerCount.set(allBrokerIds.size());
+        allBrokerIds.forEach(brokerId -> {
+            availableBytesGauges.computeIfAbsent(String.valueOf(brokerId), key -> new ConcurrentHashMap<>());
+            consumedBytesGauges.computeIfAbsent(String.valueOf(brokerId), key -> new ConcurrentHashMap<>());
+        });
         log.debug("Attempting to describe logDirs");
         return toResultStage(admin.describeLogDirs(allBrokerIds).allDescriptions())
-                .thenApply(VolumeSource::onDescribeLogDirComplete);
+                .thenApply(this::onDescribeLogDirComplete);
     }
 
-    private static VolumeUsageResult onDescribeLogDirComplete(Result<Map<Integer, Map<String, LogDirDescription>>> logDirResult) {
+    private VolumeUsageResult onDescribeLogDirComplete(Result<Map<Integer, Map<String, LogDirDescription>>> logDirResult) {
         if (logDirResult.isFailure()) {
             return failure(VolumeSourceObservationStatus.DESCRIBE_LOG_DIR_ERROR, logDirResult.getThrowable());
         } else {
             return onDescribeLogDirSuccess(logDirResult.value);
         }
     }
 
-    private static VolumeUsageResult onDescribeLogDirSuccess(Map<Integer, Map<String, LogDirDescription>> logDirsPerBroker) {
+    private VolumeUsageResult onDescribeLogDirSuccess(Map<Integer, Map<String, LogDirDescription>> logDirsPerBroker) {
         if (log.isDebugEnabled()) {
             log.debug("Successfully described logDirs: " + logDirsPerBroker);
         }
-        return success(logDirsPerBroker.entrySet()
+        List<VolumeUsage> volumeUsages = logDirsPerBroker.entrySet()
                 .stream()
                 .flatMap(VolumeSource::toVolumes)
                 .filter(Objects::nonNull)
-                .collect(Collectors.toUnmodifiableList()));
+                .collect(Collectors.toUnmodifiableList());
+
+        activeLogDirsCount.set(volumeUsages.size());
+
+        volumeUsages.forEach(volumeUsage -> {
+            availableBytesGauges.get(volumeUsage.getBrokerId())
+                    .computeIfAbsent(volumeUsage.getLogDir(), buildCounter(volumeUsage, "available_bytes"))
+                    .set(volumeUsage.getAvailableBytes());
+            consumedBytesGauges.get(volumeUsage.getBrokerId())
+                    .computeIfAbsent(volumeUsage.getLogDir(), buildCounter(volumeUsage, "consumed_bytes"))
+                    .set(volumeUsage.getConsumedSpace());
```

Comment:
why these two metrics have a different naming scheme compared to the others? (they are not camelcase and using underscore to separate)

Reply:
Hmmm thats a good question.

There ones followed the [prometheus naming convention](https://prometheus.io/docs/practices/naming/) rather than Java conventions. 

Fixed in 56e76cd9bbe5db6f04dcdab4dd805a779090c507

Reply:
Yeah, we are exposing these metrics as Java beans, right? Not via Prometheus. So better keeping the Java conventions.